src/nanoemoji/nanoemoji.py

```diff
@@ -210,8 +210,6 @@ def _chrome_command() -> str:
 def write_config_preamble(nw, font_config: FontConfig):
     if font_config.has_bitmaps:
         # set height only, let width scale proportionally
-        # for CBDT width can fairly easily overrun tight pixel size limits
-        # but there isn't much we can do about it
```

Comment:
you could leave this comment in -- even if there isn't much we can do about it, at least it attests that we are aware of the limitation.
I suppose the only feasible solution in these cases would be to reduce the --bitmap_resolution until it works..

Reply:
I'm going to add an explicit check to make_cbdt_table so you get an actual error